tool/teleport/common/teleport.go

```diff
@@ -142,12 +142,12 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 		StringVar(&ccf.NodeName)
 	start.Flag("config",
 		fmt.Sprintf("Path to a configuration file [%v]", defaults.ConfigFilePath)).
-		Short('c').ExistingFileVar(&ccf.ConfigFile)
+		Short('c').StringVar(&ccf.ConfigFile)
```

Comment:
If we don't like the way that kingpin presents a file doesn't exist error, maybe we should change kingpin.

The proper way to have a flag that should point to an existing file is to use kingpin's `ExistingFile` option. If you work around that in a few select places where you don't like the error, nothing prevents people from continuing to use `ExistingFileVar` in other parts of the code base or in future updates.